WcaOnRails/spec/helpers/application_helper_spec.rb

```diff
@@ -32,7 +32,14 @@
     it "handles multiple phrases correctly highlighting them" do
       text = "A long text with a super word in the middle (directly here) followed by the rest of an awesome and peculiar sentence. What do you think?"
       expected = "...t with a super word in the middle (directly here) <strong>followed</strong> by the rest of an awesome and peculiar sentence. What do you <strong>think</strong>?"
-      expect(helper.wca_excerpt(text, ["followed", "think"])).to eq expected
+      expect(helper.wca_excerpt(text, %w(followed think))).to eq expected
+    end
+
+    it "is case insensitive" do
```

Comment:
Philosophy thing: I probably would have folded this test into the previous test. I suppose it's a tradeoff between lots of small, simple tests, and paranoia about test runtime. I'm okay with whatever you prefer here.